### PR TITLE
New package: wlrobs-v1.0

### PR DESCRIPTION
**Pull request by Noah-Huppert**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by Noah-Huppert** [comment]
> Pull Requests become stale 90 days after last activity and are closed 14 days after that.  If this pull request is still relevant bump it or assign it.

Given the known maintainer shortage and occasional merge delays, maybe this stale bot doesn't add value?

File: srcpkgs/wlrobs/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'wlrobs'
+pkgname=wlrobs
+version=v1.0
```
Comment (classabbyamp) [inline srcpkgs/wlrobs/template:3]: remove the v here

File: srcpkgs/wlrobs/template [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+revision=1
+build_style=meson
+hostmakedepends="cmake pkg-config"
```
Comment (classabbyamp) [inline srcpkgs/wlrobs/template:6]: why does this need cmake?

File: srcpkgs/wlrobs/template [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+makedepends="wayland-devel obs-devel libX11-devel"
+short_desc="OBS plugin to screen capture wlroots based wayland compositors"
```
Comment (paper42) [inline srcpkgs/wlrobs/template:8]: What's the difference between this plugin and a proper setup with pipewire and portals?

**Comment by Noah-Huppert** [comment]
> Pull Requests become stale 90 days after last activity and are closed 14 days after that.  If this pull request is still relevant bump it or assign it.

.

**Pull request closed without merging**